R/create_figures_doc.R

```diff
@@ -17,12 +17,31 @@ create_figures_doc <- function(resdir = NULL,
                                year = NULL,
                                subdir = NULL,
                                include_all = TRUE) {
+
   model <- match.arg(model, several.ok = FALSE)
 
+  # import pre-written captions and alt text
   captions_alttext <- utils::read.csv(
     system.file("resources", "captions_alttext.csv", package = "asar")
   )
 
+  # import converted model output
+  output <- utils::read.csv(
+    paste0(resdir, "/", model_results, ".csv")
+  )
+
+  # output <- utils::read.csv(
+  #   "petrale_convert_output.csv"
+  # )
+
+  # extract quantities
+  # using the <<- exports the object to the R environment!
+  start_year <<- as.numeric(output[5,5])
```

Comment:
I would suggest not to hard code these locations as they won't necessarily always be there with all convertions

Reply:
Just coded that as an example. It's not an actual year (I don't think). I was troubleshooting where my errors were coming from and thought I'd try to minimize those sources by making my two test parameters really simple extractions from the model results df